cmd/CMakeLists.txt

```diff
@@ -14,11 +14,13 @@ function(add_cmd CMD_SRC IS_GUI)
         $<IF:$<BOOL:${IS_GUI}>,mrtrix::gui,mrtrix::headless>
         mrtrix::exec-version-lib
     )
-    set_target_properties(${CMD_NAME} PROPERTIES 
+    set_target_properties(${CMD_NAME} PROPERTIES
```

Comment:
We probably should version information as well here. Something like `MACOSX_BUNDLE_BUNDLE_VERSION ${PROJECT_VERSION}` should work. Additionally, we may want to add `MACOSX_BUNDLE_GUI_IDENTIFIER` too (which should be done automatically only after Qt 6.1).

Reply:
Done